client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx

```diff
@@ -53,12 +53,42 @@ const PatternAssemblerPreview = ( { header, sections = [], footer }: Props ) =>
 
 	useEffect( () => {
 		setScrollToSelector( null );
```

Comment:
I checked the branch and it is working as expected. One question, could we remove this `useEffect` for header?

Reply:
The selector must be cleaned when a header is chosen because if not the preview will scroll again.